### PR TITLE
Console POM cleanup

### PR DESCRIPTION
**Pull request by lorthirk**
Hi all,

Using IntelliJ, that automatically configures the projects according to the POM file, I noticed that the console one needed to be cleaned up a bit.

Please review these changes and test them with any running configuration you need!

Cheers

### CODE REVIEW

File: console/pom.xml [hunk 1]
```diff
@@ -271,7 +271,7 @@
     </dependencies>
 
     <build>
-        <finalName>console</finalName>
+        <finalName>Kapua</finalName>
```
Comment (ctron) [inline console/pom.xml]: I think this will create a war file named `Kapua.war`, right?

Comment (lorthirk) [inline console/pom.xml]: Yes, it will. Do you think it could be a problem?

**Comment by lorthirk** [comment]
As discussed on our last community meeting on June 1st, probably the best way is to rename the GWT module from "Kapua" to something more meaningful. "console" wouldn't be a good fit since GWT would then overwrite the native JavaScript "console" object. "Console" (with capital C) would probably work as a workaround, but I don't like it since it would be confusing. 

My vote is for "admin", since it is the Kapua Administration Console, and other web applications often use the word for such usages.

**Comment by ctron** [comment]
+1 for admin

**Comment by lorthirk** [comment]
So, I made some tests and both Vagrant and Docker seems to be fine, like the target folder. For me it's ready to be merged.

**Comment by Coduz** [comment]
Demo vagrant machine is broken (broker does not start) due to changes not related to this branch. 
Console works fine
We can merge this 😄

**Pull request merged**